status-payloads-spec.md

```diff
@@ -149,10 +141,91 @@ All incoming messages can be matched against a chat. Below you can find a table
 
 <!-- TODO: "group-user-message" is not complete. Does it require to explicitly join the group chat? Is there a way to invite someone? Also, if I start a new group chat (or join an existing one), I need to somehow calculate this chatID by myself. How to do it? -->
 
+### Contact Requests
+
+These messages are used to notify the receiving end that it has been added to the sender's contact. They are identified by the transit tags `c2`, `c3`, `c4` respectively, but they are all interchangeable, meaning a client SHOULD handle them in exactly the same way. 
+
+#### Payload
+
+Payload is a struct (a compound data type) with the following fields (order is important):
+
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | name | `string` | The self-assigned name of the user (DEPRECATED) |
+| 2 | profile image | `string` | The base64 encoded profile picture of the user |
+| 3 | address | `string` | The ethereum address of the user <!-- Why do we need this? can it be computed from the pk? --> |
+| 4 | fcm-token | `string` | The FCM Token used by mobile devices for push notifications (DEPRECATED) |
+| 5 | device-info | `[struct { id string, fcm-token string }]` | A list of pair `installation-id`, `fcm-token` for each device that is currently paired |
+
+#### Contact update
+
+A client SHOULD send a `ContactUpdate` to all the contacts each time:
```

Comment:
Is payload the same as described above?

Reply:
Yes, `ContactUpdate`/`ContactRequest`/`ContactRequestConfirmed` have exactly the same payload, but different transit tag (we should probably just have a single message)

Reply:
Should we make it more explicit?

Reply:
done!